my-app/src/HW37/buttonBurger.module.css

```diff
@@ -0,0 +1,29 @@
+.div {
```

Comment:
common code structure
--index.tsx (or hw-37.tsx in your case)
/Button
--Button.tsx
--Button.css
/Tabs
--Tabs.tsx
--Tabs.css